src/python/pants/backend/shell/lint/shellcheck/rules_integration_test.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from textwrap import dedent
+
+import pytest
+
+from pants.backend.shell.lint.shellcheck.rules import ShellcheckFieldSet, ShellcheckRequest
+from pants.backend.shell.lint.shellcheck.rules import rules as shellcheck_rules
+from pants.backend.shell.target_types import ShellLibrary
+from pants.core.goals.lint import LintResult, LintResults
+from pants.core.util_rules import external_tool, source_files
+from pants.engine.addresses import Address
+from pants.engine.target import Target
+from pants.testutil.rule_runner import QueryRule, RuleRunner
+
+
+@pytest.fixture
+def rule_runner() -> RuleRunner:
+    return RuleRunner(
+        rules=[
+            *shellcheck_rules(),
+            *external_tool.rules(),
+            *source_files.rules(),
+            QueryRule(LintResults, [ShellcheckRequest]),
+        ],
+        target_types=[ShellLibrary],
+    )
+
+
+GOOD_FILE = "# shellcheck shell=bash\necho 'shell known'\n"
+BAD_FILE = "echo 'shell unknown'\n"
+
+
+def run_shellcheck(
+    rule_runner: RuleRunner, targets: list[Target], *, extra_args: list[str] | None = None
+) -> tuple[LintResult, ...]:
+    rule_runner.set_options(
+        ["--backend-packages=pants.backend.shell.lint.shellcheck", *(extra_args or ())],
+        env_inherit={"PATH", "PYENV_ROOT", "HOME"},
```

Comment:
The need for PYENV_ROOT is not obvious. Is it actually needed here?

Reply:
Bad copy pasta. Thanks!